### PR TITLE
changed test_requirement.txt removed one line

### PR DESCRIPTION
**Pull request by ANA-POTJE**
## Description

Fixing the Github issue #119 Remove trufflehog pinned dependency in test_requirements.txt

## Checklist

- [ ] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes

## Legal notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

- I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
- I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorised to submit this contribution on behalf of the original creator(s) or their licensees.
- I certify that the use of this contribution as authorised by the Apache 2.0 license does not violate the intellectual property rights of anyone else.

### CODE REVIEW

File: package/test_requirements.txt [hunk 1]
```diff
@@ -10,4 +10,3 @@ requests-mock>=1.6.0, <2.0
 flake8>=3.5, <4.0
 isort
 trufflehog>=2.0.99, <3.0
```
Comment (921kiyo) [inline package/test_requirements.txt]: Could you also update the lower bound of trufflehog to be 
`trufflehog>=2.1.0, <3.0` ?

Comment (921kiyo) [inline package/test_requirements.txt]:
```suggestion
trufflehog>=2.1.0, <3.0
```

Comment (ANA-POTJE) [reply package/test_requirements.txt]: Thanks Kiyo! Some questions ...
(1) " .. update RELEASE.md with your name and Github link please? .."
I believe I should change RELEASE.md, then again: "git commit / git push / create a new PR, am I correct? 

(2) " ... add the description of which Github issue you are addressing in the PR ..."
Done! Please check ...

(3) " .. Could you also update the lower bound of trufflehog to be trufflehog>=2.1.0, <3.0 ? .."
Is this a new request, or it is inside the same Github issue?
Should I update RELEASE.md, update the lower bound of trufflehog and then "git commit / git push ..."?

Comment (ANA-POTJE) [reply package/test_requirements.txt]: Sorry, one more question … Where exactly in the RELEASE.md file should I include my name and GitHub link please? Should I include a section in the beginning of the file such as:
## supporting contributions
[Ana Potje](https://github.com/ANA-POTJE)
Thanks!

Comment (921kiyo) [reply package/test_requirements.txt]: @ANA-POTJE 
You don't need to create a new PR. You can update RELEASE.md and lower bound of trufflehog (I suggested the change below, which you can accept), then "git commit/ git push" in this PR.
 
For the RELEASE.md, see how the previous contributors added their name in https://github.com/quantumblacklabs/kedro/blob/develop/RELEASE.md  

Let me know if anything is not clear :)

Comment (921kiyo) [reply package/test_requirements.txt]: For FYI, see how the person updated the RELEASE.md in a PR.  https://github.com/quantumblacklabs/kedro/pull/93/files

**Comment by ANA-POTJE** [comment]
Hi there, changes were done (included my name in RELEASE.md updated the lower bound of trufflehog to be trufflehog>=2.1.0, <3.0, as requested.
I managed to run a git commit successfully.
But I got an error in my git push … possibly because my local version cloned from kedro-viz is outdated (?), would you please kindly guide me? Thanks!!!

**Comment by ZainPatelQB** [comment]
Hey @ANA-POTJE, thanks for this. It all seems fine here, so your `git push` seems to have worked. If you're getting an error, you might try `git pull` before `git push` but it's hard to say without the error.

Could you also please tick the legal notice box in the PR so we can merge your PR in? :)

**Comment by ANA-POTJE** [comment]
Hi Zain, "legal notice" box is ticked … I hope all ready for merge now? ;)

**Comment by ZainPatelQB** [comment]
Merged. :)

**Comment by ANA-POTJE** [comment]
That's great!!! Very happy to help! :)))

**Pull request merged**